src/store/modules/user/actions.ts

```diff
@@ -83,7 +82,7 @@ const actions: ActionTree<UserState, RootState> = {
   async getProfile ( { commit, dispatch }) {
     const resp = await UserService.getProfile()
     if (resp.status === 200) {
-      const localTimeZone = moment.tz.guess();
+      const localTimeZone = DateTime.now().zoneName;
```

Comment:
We haven't set the default Luxon timezone
`Settings.defaultZone = this.userProfile.userTimeZone;`

Reply:
Also, when we are setting the timezone
https://github.com/hotwax/receiving/pull/179/files#diff-d3c10a6ece79180d52468851ebe745d66365adafc59ee774a50b5687b3d42ff6R128